packages/react-ui/src/features/flows/components/flow-table.tsx

```diff
@@ -99,32 +43,221 @@ const filters: DataTableFilter[] = [
   },
 ];
 
-async function fetchData(queryParams: URLSearchParams) {
-  return flowsApi.list({
-    projectId: authenticationSession.getProjectId(),
-    cursor: queryParams.get('cursor') ?? undefined,
-    limit: parseInt(queryParams.get('limit') ?? '10'),
-    status: (queryParams.getAll('status') ?? []) as FlowStatus[],
-  });
-}
-
 const FlowsTable = () => {
   const navigate = useNavigate();
+  const [isRenameDialogOpen, setIsRenameDialogOpen] = useState(false);
+  const [isShareDialogOpen, setIsShareDialogOpen] = useState(false);
+  const [isImportDialogOpen, setIsImportDialogOpen] = useState(false);
+  const [refresh, setRefresh] = useState(0);
+  const [selectedFlow, setSelectedFlow] = useState<{
+    flowId: string;
+    flowName: string;
+    flowVersion?: FlowVersion;
+  }>({
+    flowId: '',
+    flowName: '',
+    flowVersion: undefined,
+  });
+
+  const { refetch } = flowsHooks.useFlows();
+
+  async function fetchData(queryParams: URLSearchParams) {
+    return flowsApi.list({
+      projectId: authenticationSession.getProjectId(),
+      cursor: queryParams.get('cursor') ?? undefined,
+      limit: parseInt(queryParams.get('limit') ?? '10'),
+      status: (queryParams.getAll('status') ?? []) as FlowStatus[],
+    });
+  }
+
+  const { mutate: createFlow, isPending: isCreateFlowPending } = useMutation<
+    PopulatedFlow,
+    Error,
+    void
+  >({
+    mutationFn: async () => {
+      const flow = await flowsApi.create({
+        projectId: authenticationSession.getProjectId(),
+        displayName: 'Untitled',
+      });
+      return flow;
+    },
+    onSuccess: (flow) => {
+      window.open(`/flows/${flow.id}`, '_blank', 'noopener');
+      refetch();
```

Comment:
Remove this, It's not neeeded and change the statle to 0 in useFlows, Let's not handle the caching, It will be probelmatic and this open new windows anyway :D